cache_run_c_code = 'Test does not build: Traceback (most recent call last):\n  File "/home/pradeepkumar/Desktop/ns3-plc-with-gnosis/ns-3.15/.waf-1.6.11-30618c54883417962c38f5d395f83584/waflib/Tools/c_config.py", line 445, in run_c_code\n    bld.compile()\n  File "/home/pradeepkumar/Desktop/ns3-plc-with-gnosis/ns-3.15/.waf-1.6.11-30618c54883417962c38f5d395f83584/waflib/Build.py", line 190, in compile\n    raise Errors.BuildError(self.producer.error)\nBuildError: Build failed\n -> task in \'testprog\' failed (exit status 1): \n\t{task 140671363733072: cxx test.cpp -> test.cpp.1.o}\n[\'/usr/bin/g++\', \'-std=c++0x\', \'-DNS3_ASSERT_ENABLE\', \'-DNS3_LOG_ENABLE\', \'../test.cpp\', \'-c\', \'-o\', \'test.cpp.1.o\']\n'
